src/atomics/poly.jl

```diff
@@ -1,27 +1,39 @@
-
-function poly(scene::makie, points, attributes::Dict)
-    last(points) == first(points) && pop!(points)
-    polys = GeometryTypes.split_intersections(points)
-    result = []
-    for poly in polys
-        mesh = GLNormalMesh(poly) # make polygon
-        if !isempty(GeometryTypes.faces(mesh)) # check if polygonation has any faces
-            push!(result, GLVisualize.visualize(mesh, Style(:default), kw_args))
-        else
-            warn("Couldn't draw the polygon: $points")
-        end
+function _poly(scene::makie, attributes::Dict)
+    attributes = poly_defaults(scene, attributes)
+    println(typeof(attributes[:positions]))
+    bigmesh = lift_node(attributes[:positions]) do p
```

Comment:
Any guidance on when `lift_node` is used? (I will contribute to docs.) This is one of the things I see everywhere in the GL* packages but don't really understand.

Reply:
it's basically the same as Reactive.map(f, signal) and just registers a callback that gets called whenever 'signal' updates... it will return a new signal which holds 'f(value(signal))'